Vagrantfile

```diff
@@ -31,5 +31,6 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provision "shell", path: "provisioning/Build.ps1", args: "-originalBuildScriptPath \"C:\\vagrant\\provisioning\\\""
+  # XXX uncomment once testing in CI concludes
```

Comment:
What event are we waiting for?

Reply:
The time for this to be reverted has come as I've concluded work on [CI test failures](https://issues.gpii.net/browse/GPII-3528).
The idea behind this was to make CI builds faster by excluding the creation of installer step.